src/v2/views/MonsterCollectionOverlay/base.tsx

```diff
@@ -0,0 +1,214 @@
+import OverlayBase from "src/v2/components/core/OverlayBase";
+import { createTheme, styled } from "src/v2/stitches.config";
+
+import background from "src/v2/resources/collection/bg.png";
+import formImg from "src/v2/resources/collection/deposit-bg.png";
+import activationButtonImg from "src/v2/resources/collection/button-activation.png";
+import activationButtonHoverImg from "src/v2/resources/collection/button-activation-over.png";
+import disabledButtonImg from "src/v2/resources/collection/button-inactive.png";
+import cancelButtonImg from "src/v2/resources/collection/button-cancel.png";
+import cancelButtonHoverImg from "src/v2/resources/collection/button-cancel-over.png";
+import rewardImg from "src/v2/resources/collection/reward-bg.png";
+import dotImg from "src/v2/resources/collection/dot.png";
+import darkTextBg from "src/v2/resources/collection/dark-text-bg.png";
+import itemBg from "src/v2/resources/collection/item-bg.png";
+
+export const theme = {
```

Comment:
I tried to use [the theme feature of Stitches](https://stitches.dev/docs/theming#add-a-new-theme) but it caused some flickering when the overlay got mounted, so I reverted it to a simple object.

I have some mixed feelings about how this theme object should be organized, but I couldn't come up with something consistent. Some elements don't use this theme object at all and the colour and gradients are different by a bit, making the effort to standardize the tokens bit useless.